src/angular/table/sort/sort-header.scss

```diff
@@ -1,5 +1,7 @@
 @use '@sbb-esta/lyne-elements' as sbb;
 
+$sbb-sort-animation-transition: 225ms cubic-bezier(0.4, 0, 0.2, 1);
```

Comment:
should we also consider the lyne variable for timing and disabling animation? I think it's a good idea :). maybe we anyways, one day, move more logic of the sort header to the lyne-elements

Reply:
Do you mean remove the variable and replace its usage with `sbb-animation-duration-5x/6x` and `sbb-animation-easing`?

Reply:
yes, like I suggested below, expect that I forgot the --sbb-disable-animation-duration variable to consider